app/views/stories/_listdetail.html.erb

```diff
@@ -129,8 +129,11 @@ class="story <%= story.current_upvoted? ? "upvoted" : "" %>
 
         <% if story.is_editable_by_user?(@user) %>
           <span> | </span>
-          <a href="<%= edit_story_path(story.short_id) %>" class="<%=
-            story.has_suggestions? ? "story_has_suggestions" : "" %>">edit</a>
+          <a href="<%= @user.is_moderator? ? edit_moderate_story_path(story.short_id) : edit_story_path(story.short_id) %>"
```

Comment:
So I know this must sound odd, but I would like mods to see two edit links. So this should always be the `edit_story_path` link. Below, you added a `if @user.try(:is_moderator?)` to link to the mod notes for the user (handy!). Please add a second `edit` link leading to the new moderator story editing controller.

I am slowly working towards making it so mods always see the same UI as users, then also additional, clearly marked (#1259) mod UI. There has been minor but avoidable confusion about things because mods can't remember exactly what users see for scores, vote summaries, etc.